requirements.pip

```diff
@@ -1,8 +1,8 @@
-botocore==1.12.102
-awscli==1.16.112
+botocore==1.12.156
+awscli==1.16.166
```

Comment:
Small typo in the PR title, which could be misleading in the future: awscli bumped to 1.16.166 and sceptre to 2.1.3